modules/ROOT/pages/guide-data-modeling.adoc

```diff
@@ -6,9 +6,6 @@
 :tags: graph-modeling, data-model, schema, model-process, model-domain
 :description: This guide is designed to walk you through the graph data modeling lifecycle of Neo4j. You will be introduced to the basic process of designing a graph data model that can answer a wide range of business questions across a variety of domains.
 
-.Goals
```

Comment:
why did you remove the description? what's in the asciidoc variable only gets set as html-meta-description for google, not for the user.